### PR TITLE
dded popp/wimmer pics, updated program

### PR DESCRIPTION
**Pull request by alexander-diab**

### CODE REVIEW

File: _i18n/de/summit2023.md [hunk 1]
```diff
@@ -53,14 +57,14 @@ Felix Kronlage-Dammers, Product Owner SCS IaaS
 * "Federation Services – Die Open Source Werkzeugkiste zum Aufbau souveräner Datenökosysteme" (DE)
 Andreas Weiss, Geschäftsbereichsleiter Digitale Geschäftsmodelle, eco - Verband der Internetwirtschaft e.V.
 
-* Impulsvortrag: tbd (DE)
+* Impulsvortrag: "Die strategische Relevanz von SCS als Praxisbeispiel digitaler Transformation" (DE)
```
Comment (fkr) [inline _i18n/de/summit2023.md:60]: Sometimes we use " " and sometimes „ " (see lines 60 and 66). Imho we should be consistent and use „ " for german.

File: _i18n/en/summit2023.md [hunk 2]
```diff
@@ -19,14 +19,18 @@ Day 1:
 Ernst Stöckl-Pukall, Director Department Digitisation & Industrie 4.0
 
 * Panel discussion: "Why digital sovereignty is important for Europe" (DE)
-Dr. Franziska Brantner, Parliamentary State Secretary at the Federal Ministry for Economic Affairs and Climate Action of Germany;
-Dr.Reinhard Brandl, Member of Parliament;
-Stephan Ilaender, STACKIT;
-Jan Hill, Director Public Services, adesso SE
+Dr. Franziska Brantner, Parliamentary State Secretary at the Federal Ministry for Economic Affairs and Climate Action of Germany;  
+Dr.Reinhard Brandl, Member of Parliament;
```
Comment (fkr) [inline _i18n/en/summit2023.md:23]:
```suggestion
Dr. Reinhard Brandl, Member of Parliament;  
```

**Comment by alexander-diab** [comment]
I have added Martin Wimmers name and the title of his talk - german and english

File: _i18n/en/summit2023.md (hunk 2 above)
Comment (alexander-diab) [reply _i18n/en/summit2023.md:23]: lgtm!

File: _i18n/de/summit2023.md (hunk 1 above)
Comment (alexander-diab) [reply _i18n/de/summit2023.md:60]: Yes, but this program will be replaced probably by tomorrow. I would not put too much effort into the form right now

**Review by maxwolfs — APPROVED** [review]
lgtm

**Pull request merged**